### DISCUSSION TITLE
Deploying to ECS Fargate

### DISCUSSION THREAD

**Discussion by moisish** [serversideup/docker-php#189] [Q&A]
I am deploying my container to ECS Fargate but I keep getting an error for the container that will act as a webserver (nginx & fpm)

```
s6-overlay-suexec: fatal: can only run as pid 1
```

The same image runs fine when I override the command to run Laravel horizon

**Comment by jaydrogers** [comment]
I know this is something that happens when the underlying system is not allowing the container to use "Process ID #1".

This is an issue on Fly.io and sounds the same with ECS fargate.

Unfortunately, this might be an upstream issue with ECS Fargate where they need to allow this behavior. Our images are based off of S6 Overlay (https://github.com/just-containers/s6-overlay), so publishing a post there might get better support. Feel free to tag me on it if you want to CC me on the thread.

**Reply by moisish** [reply]
If I provide a custom command to the container I don't have any issues with it tho. So it's only my nginx container that cannot run :/

**Reply by jaydrogers** [reply]
Hmm... I wonder if there is someone you can reach out to on ECS Fargate's side?

I know with Fly.io, I was talking to one of their main sysadmins and he said it was a change they would have to make on their side.

**Comment by jaydrogers** [comment]
Just an update. I am closing this issue for a consolidated discussion on this issue here. Any further updates, please comment on this thread: https://github.com/serversideup/docker-php/discussions/191
